### ISSUE TITLE
Ayaneo air 1s Gyro flipped

### ISSUE THREAD

**Issue by dansl** [ShadowBlip/InputPlumber#217]
When using a configuration that enables Gyro features (steam deck or DS-edge), the pitch and yaw are flipped.

**Comment by uejji** [comment]
Hello again.

Can you try adding a mount_matrix to your config file?  See https://github.com/ShadowBlip/InputPlumber/blob/6d2c65fb0c66bcdf79e376232fa28c4b9aedbf0d/rootfs/usr/share/inputplumber/devices/50-ayaneo_air_plus.yaml#L53 as an example.

**Comment by dansl** [comment]
That fixed it! 

But... Only fixed it for games that have in-game gyro support (ex: Horizon Zero Dawn). I notice that when I look at the Steam calibration test page Y and Z are flipped...

**Comment by uejji** [comment]
You'll probably need to adjust the signs and value positions of the mount_matrix until you find the right combination.  Just make sure each row has only one 1/-1 and two 0s, and that each row has the 1/-1 in a different column.  Unfortunately I can't provide much more specific help than that.

**Comment by dansl** [comment]
Yes, the original combination is correct in a game, but when I view the calibration tools in steam, they are incorrect. If I change the values to fix steam, then it breaks in the game controls. So the game sees it properly, but steam does not for some reason... (specifically I am using SteamFork OS)

**Comment by uejji** [comment]
Is this only with the Steam Deck driver or is it also occurring with the DS5 Edge driver?

**Comment by uejji** [comment]
After asking around and doing some of my own testing, if the in-game performance is as you expect, then it's probably just a visual bug on the calibration screen right now.

**Comment by dansl** [comment]
I was testing with the Steam Deck driver, but if I switch to DS-edge it is now the opposite. Steam reports correct functionality, but the game is not correct...

**Comment by uejji** [comment]
Moving forward, the intended result will be that gyro output matches Steam Deck.  You may have to modify your game settings if results are not expected.  See #220
